src/index.js

```diff
@@ -15,7 +15,10 @@ const imgPropTypes = {
   unloader: node,
   decode: bool,
   src: oneOfType([string, array]),
-  container: func
+  container: func,
+  imgContainer: func,
```

Comment:
I'm not quite sure what you mean here...

I thought the idea of having the 3 optional `img/loader/unloaderContainer` was that those override the `container` prop. That is, if using a prefixed prop, it doesn't affect the others, and the `container` affects them all (if the relevant prefixed prop isn't set). And with the default no-op `container` the code `loaderContainer || container || imgContainer` will never use `imgContainer`, as it will always stop before it. But I think that would be the desired and intuitive/expected behavior.

But am I understanding you correctly?

(I was thinking about only having the two `un/loaderContainer`s, and so to use a container for only the image but not the loaders, you would set `container`, and then set the `un/loaderContainer`s to a no-op. But maybe it's more intuitive having `imgContainer` also, and so to get the same behavior you only need to set `imgContainer`.)

Reply:
Hmm - I hear your argument and was going back and forth on this myself.

I think that for now, we should _not_ introduce a new type and instead continue using `container`, and default the new `(un)loaderContainer`'s to `container`. Ultimately, this is how things work today. Should people request absolute control over this, we could add the `imgContainer` in a future release.